src/api/getFetch.js

```diff
@@ -0,0 +1,7 @@
+export const getFetch = async () => {
+  const response = await fetch(
+    "https://bootcamp-api.codeit.kr/api/sample/folder"
```

Comment:
```suggestion
const API_BASE_URL = 'https://bootcamp-api.codeit.kr' // API_BASE_URL은 함수 밖으로
    `${API_BASE_URL }/api/sample/folder`
```
이렇게 하셔서 API_BASE_URL을 재사용 하는 것을 권장드립니다.